converter/src/main/java/gov/cms/qpp/conversion/decode/placeholder/DefaultDecoder.java

```diff
@@ -40,10 +42,11 @@ protected DecodeResult internalDecode(Element element, Node thisnode) {
 	}
 
 	public static void removeDefaultNode(List<Node> nodes) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use 'java.lang.Iterable' here; it is a more general type than 'List'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3242)